src/test/resources/data/StudentProfilePageUiTest.json

```diff
@@ -37,13 +37,43 @@
       "isInstructor": false,
       "email": "SHomeUiT.benny.c@gmail.tmt",
       "institute": "TEAMMATES Test Institute 1"
+    },
+    "instructor1OfTestingSanitizationCourse": {
```

Comment:
"account" is different from "instructors" and "students". Therefore, I don't think we should name the account as "instructor1OfTestingSanitizationCourse"